src/gui/advancedsettings.cpp

```diff
@@ -453,7 +456,13 @@ void AdvancedSettings::loadAdvancedSettings()
     addRow(HASHING_THREADS, (tr("Hashing threads") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#hashing_threads", "(?)"))
             , &m_spinBoxHashingThreads);
 #endif
-
+    // Max active rechecking torrents
+    m_spinBoxMaxActiveChecking.setMinimum(-1);
+    m_spinBoxMaxActiveChecking.setMaximum(std::numeric_limits<int>::max());
+    m_spinBoxMaxActiveChecking.setValue(session->maxActiveChecking());
+    m_spinBoxMaxActiveChecking.setSpecialValueText(tr("∞"));
```

Comment:
There is predefined constant in "unicodestrings.h". And it shouldn't be translatable, should it?